types/stripe/index.d.ts

```diff
@@ -7336,6 +7336,12 @@ declare namespace Stripe {
              */
             source_transaction?: string;
 
+            /**
+             * The source balance to use for this transfer. One of bank_account
+             * or card. For most users, this will default to card.
+             */
+            source_type?: 'bank_account' | 'card';
```

Comment:
This is inconsistent with `transfers.ITransfer` which uses `transfers.SourceTypes`.  That type includes `alipay_account` and `bitcoin_receiver`.  Unless there's an announcement that they're no longer supported they should probably be included.

Reply:
Thanks! Made the change 👍